src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/tools/monitoring/outputMonitor.ts

```diff
@@ -438,16 +437,10 @@ export class OutputMonitor extends Disposable implements IOutputMonitor {
 		const prompt = confirmationPrompt.prompt;
 		const options = confirmationPrompt.options;
 
-		const currentMarker = this._execution.instance.registerMarker();
-		if (!currentMarker) {
-			// Unable to register marker, so cannot track prompt location
-			return undefined;
-		}
-		if (this._lastPromptMarker?.line === currentMarker.line) {
+		if (this._lastPrompt?.includes(prompt) || this._lastPrompt && prompt?.includes(this._lastPrompt)) {
```

Comment:
No understanding the problem mentioned in https://github.com/microsoft/vscode/issues/263888, in that scenario, the marker should be pointing at line 3 for the first one and line 9 for the second?